extension/src/test/e2e/extension.test.ts

```diff
@@ -129,7 +129,7 @@ describe('Experiments Table Webview', function () {
     expect(finalRows.length).toStrictEqual(initialRows + experimentRow)
     await webview.unfocus()
     await closeAllEditors()
-    await waitForDvcToFinish(90000)
+    await waitForDvcToFinish(120000)
```

Comment:
Increased the timeout that was failing but is there a better way to solve this?

Reply:
Thank you.

This amount of time is a bit concerning (90000) already was. I hope there isn't something else going on 🤞🏻.